system/src/system_network_cellular.h

```diff
@@ -133,11 +133,15 @@ class CellularNetworkInterface : public ManagedIPNetworkInterface<CellularConfig
     bool has_credentials() override
     {
         bool rv = cellular_sim_ready(NULL);
-        LOG(INFO,"%s", (rv)?"Sim Ready":"Sim not inserted? Detecting...");
+        if (rv) {
+            LOG(INFO,"Sim Ready");
+        }
```

Comment:
Reporting only `Sim ready`. 
When sim is not ready and AT is working, it will be anyway reported here - https://github.com/particle-iot/device-os/blob/47bac6ae62977435096b33341aad0ee20d036ed8/hal/src/electron/modem/mdm_hal.cpp#L848